.github/workflows/pr-precommit.yml

```diff
@@ -19,9 +19,8 @@ name: 'Use pre-commit to validate Pull Request'
 on:
   pull_request:
     types:
-    - edited
     - opened
-    - labeled
```

Comment:
Wouldn't this leave the label validation in the failed state if a label was added after the PR was created (i.e. label wasn't added during PR creation)?

I'm unsure if it was the "labeled" or "edited" event that triggers when a label is added or removed.

Reply:
I think you are reviewing this file with the label validation in mind. The existence of a label does not change anything about whether the `pre-commit` validation passes.

Reply:
I've added links to the definitions of each event in the description. The drop-down selector for each event was not obvious (to my eyes).

Reply:
Ohh I'm sorry, I did read the pr-precommit file with label validation in mind.  Looks good to me.